apps/fipsinstall.c

```diff
@@ -81,6 +82,41 @@ const OPTIONS fipsinstall_options[] = {
     {NULL}
 };
 
+typedef struct {
+    unsigned int self_test_onload : 1;
```

Comment:
Using bitfields here is the epitome of a premature optimization. I betcha it even makes the code bigger while saving one byte in data.

Reply:
Bitfields are rare within OpenSSL. I will note that the static options structs below require comments to identify each field. Using bitwise `#define`s would make the structures more readable.